src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double length;
+
+    public Rectangle(String color, double width, double length) {
+        super(color);
+        this.width = width;
+        this.length = length;

```

Comment:
Creating a figure, don't pass expressions in the constructor directly. Create separate variables for 'width' and 'length' and pass them to the constructor for better code readability.